pkg/utils/utils.go

```diff
@@ -2,12 +2,18 @@ package utils
 
 import (
 	"fmt"
+	"math/rand"
 	"path"
 	"path/filepath"
+	"strconv"
+	"strings"
 
 	"github.com/vishvananda/netlink"
 )
 
+// Max netdevice length is 15 chars. see if.h linux uapi header IFNAMSIZ
+const localIFNAMSIZ=16
```

Comment:
you can use the const https://golang.org/pkg/syscall/ it in hex

Reply:
I originally thought that its not needed to link against it but thinking on this further, i already use netlink which imports `syscall`